app/stacks/MasterDetailStack/index.js

```diff
@@ -139,7 +139,6 @@ const ModalStackNavigator = React.memo(({ navigation }) => {
 				<ModalStack.Screen
 					name='MessagesView'
 					component={MessagesView}
-					options={MessagesView.navigationOptions}
```

Comment:
Why did you remove this from the tablet mode but keep it on the regular inside stack?

Reply:
Removed.